maps/groundbase/gb-z1.dmm

```diff
@@ -9324,19 +9363,13 @@
 /turf/simulated/floor/tiled/dark,
 /area/groundbase/civilian/bar)
 "vl" = (
-/obj/machinery/light/small,
-/obj/structure/toilet/prison{
-	dir = 4
+/obj/structure/cable/yellow{
+	icon_state = "1-8"
 	},
-/obj/machinery/button/remote/airlock{
-	dir = 4;
-	id = "z1stall4";
-	pixel_x = -27;
-	specialfunctions = 4
+/obj/effect/floor_decal/corner/paleblue{
+	dir = 10
 	},
-/obj/effect/floor_decal/rust,
-/obj/effect/decal/cleanable/dirt,
-/turf/simulated/floor/tiled/asteroid_steel,
+/turf/simulated/floor/tiled/white,
```

Comment:
As a note, the reason why the men's bathroom looks grimy in contrast to the women's bathroom is a map joke. Men's restrooms tend to look worse than women's restrooms. It was taken to a silly extreme here for the joke, considering the restrooms are ungendered on the other maps. If you're trying to fit a blue bin in, just remove the nanomed since it is rarely (if ever) used and in an emergency, the adjacent bathroom has one. 

Also the men's now has no mirrors. 
![image](https://user-images.githubusercontent.com/96669534/232267636-d203cea0-f691-4ab4-a78c-eba27c541b2c.png)

Reply:
I think joke being gone is perfectly fine personally, to note.

Reply:
I FORGOT THE MIRRORS DAMNIT. I can get that on my next pass by over the next few weeks since we are map rotating soon anyways.